cmd/root.go

```diff
@@ -1,19 +1,22 @@
 package cmd
 
 import (
+	"os"
+	"os/signal"
+	"syscall"
+
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+
 	"github.com/trustwallet/blockatlas/config"
 	"github.com/trustwallet/blockatlas/pkg/logger"
 	"github.com/trustwallet/blockatlas/platform"
 	"github.com/trustwallet/blockatlas/storage"
-	"os"
-	"os/signal"
-	"syscall"
 )
 
 var (
-	Storage = storage.New()
+	Storage *storage.Storage
+	DevMode bool
```

Comment:
why did you use global var?

Reply:
for Storage or DevMode? DevMode just because it is in the rootCmd. I can change it if you want ^^